packages/eslint-plugin-tslint/src/rules/config.ts

```diff
@@ -118,7 +119,7 @@ export default createRule<Options, MessageIds>({
     context,
     [{ rules: tslintRules, rulesDirectory: tslintRulesDirectory, lintFile }],
   ) {
-    const fileName = context.getFilename();
+    const fileName = path.resolve(context.getCwd(), context.getFilename());
```

Comment:
Explaining: with `EXPERIMENTAL_ProjectService`, relative paths might no longer match up with how the project views its root file. So switching to absolute paths is more reliable.